modules/manually-configure-iam-nutanix.adoc

```diff
@@ -0,0 +1,130 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_nutanix/configuring-iam-nutanix.adoc
+
+:_content-type: PROCEDURE
+[id="manually-create-iam-nutanix_{context}"]
+= Configuring IAM for Nutanix
+
+Installing the cluster requires that the Cloud Credential Operator (CCO) operate in manual mode. While the installation program configures the CCO for manual mode, you must specify the identity and access management secrets.
+
+.Prerequisites
+
+* You have configured the `ccoctl` binary.
+
+* You have an `install-config.yaml` file.
+
+.Procedure
+
+. Create a YAML file that contains the credentials data in the following format:
++
+.Credentials data format
+[source,yaml]
+----
+credentials:
+- type: basic_auth <1>
+  data:
+    prismCentral: <2>
+      username: <username_for_prism_central>
+      password: <password_for_prism_central>
+    prismElements: <3>
+    - name: <name_of_prism_element>
+      username: <username_for_prism_element>
+      password: <password_for_prism_element>
+----
+<1> Specify the authentication type. Only basic authentication is supported.
+<2> Specify the Prism Central credentials.
+<3> Optional: Specify the Prism Element credentials.
++
+[NOTE]
+====
+The default location for the YAML file is in `<home_directory>/.nutanix/credentials`. To specify a different directory, use the `--credentials-source-filepath` flag. This procedure uses `<path_to_credentials_file>` to refer to this directory.
+#The procedure below lacks any instance of `--credentials-source-filepath`. Where/when does a user need to specify this if they are not using the default directory?#
```

Comment:
by default we pick credentials from <home_directory>/.nutanix/credentials. If user has credentials stored at some other place they can use `--credentials-source-filepath` flag

Reply:
Ok I think adding this info to step 3 makes sense. Updated and will refresh PR shortly.